packages/xo-server-sdn-controller/package.json

```diff
@@ -30,9 +30,12 @@
   },
   "dependencies": {
     "@xen-orchestra/log": "^0.2.0",
+    "@xen-orchestra/openflow": "^0.1.0",
+    "ipaddr.js": "^1.9.1",
     "lodash": "^4.17.11",
     "node-openssl-cert": "^0.0.117",
     "promise-toolbox": "^0.15.0",
+    "util": "^0.12.3",
```

Comment:
What is this?

Reply:
I don't understant the question.

In `openflow-channel.js`, i import it
```js
import util from 'util'
```

in order to call
```js
util.inspect(msg)
```

Reply:
[`util`](https://nodejs.org/docs/latest/api/util.html) is a native module, it should not be in `package.json`.